assets/js/screens/Collection/Collection.test.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
Same here, can you add some extra tests which cover:

- Rendering of Edit / Delete buttons.
- Maybe even a test for clicking on each button, what should happen?   Is it possible to test?

Reply:
I was able to test delete button click fully. For edit button, the click event results in firing nested un-existing mock events due to the way it is rendered with RouterApollo. I was able to test that edit button exists on screen. The latter part will be covered in integration testing.